### ISSUE TITLE
Scam - You'll lose all your SOL

### ISSUE THREAD

**Issue by paschaldev** [georgebrasco/magic-eden-bot#3]
Do not use this, this line in the code is suspicious, you're going to lose all your SOL if you attempt to run this.

<img width="1045" alt="Screen Shot 2022-05-24 at 8 32 13 PM" src="https://user-images.githubusercontent.com/13323281/170117431-7af34218-5fdc-4c03-b941-559a914f114c.png">

**Comment by Ishaq34** [comment]
Yes i think it's suspect but maybe it's possible to adapt this code for a real bot

**Comment by paschaldev** [comment]
@Ishaq34 Here: https://github.com/SolanaNFTCollector/Solana-Minting-Bot/i
